src/@next/Card/Card.tsx

```diff
@@ -10,12 +10,15 @@ import {
   StyledCardHeaderSection,
   StyledCardHeaderSectionHalf,
   StyledCardHeaderWrapper,
+  StyledCustomHeader,
 } from './CardStyle';
 import { Section } from './Section';
 
+const PRIMITIVE_TYPES = ['string', 'number'];
```

Comment:
need to remove unused var, maybe should set the lint to error for this 🤔

Reply:
Deleted